### PR TITLE
Closes #1

### LINKED ISSUE #1: See All Reminders
When the user visits the root of the application (e.g. the "/" URL), they should see a list of all of the reminders on the page.

- Each reminder should have a class of .spec-reminder-item.

- There is a failing test set up in tests/acceptance/reminder-list-test.js.

Hot Tip: If you put “Closes #1” (assuming the issue is marked #1) in the pull request, your commit, or anywhere in the body of your PR, it will automatically close this issue when it successfully merged into master.

### PR DESCRIPTION
**Pull request by MilkMan90**
## Purpose

[Issue #1 ](https://github.com/turingschool-projects/1608-remember-6/issues/1)

@martensonbj @tman22 @stevekinney @nfosterky

Get reminders from Mirage store and display them on the page when navigating to the root ('/')

## Approach

Our change addresses the problem directly, and adds another test to confirm functionality with 15 reminders as well as 5.

### Learning

Before and during implementation we referenced lessons taught by super awesome Turing instructors.

[Beard Beats Repo](https://github.com/turingschool-examples/beard-beats-two)
[Ember Components](https://github.com/turingschool-examples/ember-components)

#### Blog Posts

N/A

### Open Questions and Pre-Merge TODOs

- [x] render reminder items to page
- [ ] get all tests passing by adding links for each reminder  

### Test coverage 

We added a second test to check that 15 reminders were added to the page in addition to the original 5.

### Merge Dependencies and Related Work

N/A

### Follow-up tasks

- Receive feedback for issue #1.
- Issue #2 when we get it.

### Screenshots

screen shot 2016-12-06 at 3 48 21 pm

### CODE REVIEW

File: app/models/reminder.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import DS from 'ember-data';
+
+export default DS.Model.extend({
+  title: DS.attr('string'),
+  date: DS.attr('string'),
```
Comment (Tman22) [inline app/models/reminder.js]: Do you want to use DS.attr('string')? For DATE, are there any other DS.attrs you could use? https://guides.emberjs.com/v2.10.0/models/defining-models/#toc_transforms

File: app/templates/application.hbs [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<div>
+
```
Comment (Tman22) [inline app/templates/application.hbs]: Seems like the spacing here is excessive.

**Comment by MilkMan90** [comment]
Should be fixed.

File: app/models/reminder.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import DS from 'ember-data';
+
+export default DS.Model.extend({
+  title: DS.attr('string'),
+  date: DS.attr('date'),
+  notes: DS.attr('string'),
+  pinned: false
```
Comment (stevekinney) [inline app/models/reminder.js]: I suspect that you're going to want to save this to the API at some point. So, you'd be better off with: `pinned: DS.attr('boolean', { defaultValue: false })`.

**Comment by stevekinney** [comment]
I'd also try to add the name of the feature you're implementing into the title of the PR.

**Pull request merged**